### PR TITLE
fix(cost-analysis): access to element by $el in component ref

### PR DESCRIPTION
**Pull request by WANZARGEN**
### To Reviewers
- [ ] Skip (`style`, `chore` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [x] Bug fixes
- [ ] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [x] `Error / Warning / Lint / Type`

### Description

There was critical bug in Cost Analysis pdf download.

- CostAnalysisChart component emitted 'rendered' event with parameters that includes Vue component, not HTML element. 
So I this PR fixed it to event parameters include only elements.

- To fix this bug, it was pretty hard. Error messages from PdfDownloadOverlay component was not helpful because try/catch statement was not written where it really needed to be written.
So this PR adds try/catch statement to PdfDownloadOverlay component's some functions.

### Things to Talk About

### CODE REVIEW

File: src/services/cost-explorer/cost-analysis/modules/CostAnalysisChart.vue [hunk 1]
```diff
@@ -116,7 +116,7 @@ export default {
             legends: [] as Legend[],
             chartData: [] as Array<XYChartData|PieChartData>,
             chart: null as XYChart | PieChart | null,
-            queryRef: null as null|HTMLElement,
+            queryRef: null as null|Vue,
```
Comment (yuda110) [inline src/services/cost-explorer/cost-analysis/modules/CostAnalysisChart.vue:119]: 👀

**Review by yuda110 — APPROVED** [review]
LGTM!!!

File: src/services/cost-explorer/cost-analysis/modules/CostAnalysisChart.vue (hunk 1 above)
Comment (minseolee) [inline src/services/cost-explorer/cost-analysis/modules/CostAnalysisChart.vue:119]: 👍👍👍👍👍

**Review by minseolee — APPROVED** [review]
LGTM!!!!!

**Pull request merged**